temporalio/contrib/opentelemetry.py

```diff
@@ -165,10 +176,15 @@ def _start_as_current_span(
 
     def _completed_workflow_span(
         self, params: _CompletedWorkflowSpanParams
-    ) -> _CarrierDict:
+    ) -> Optional[_CarrierDict]:
         # Carrier to context, start span, set span as current on context,
         # context back to carrier
 
+        # If the parent is missing and user hasn't said to always create, do not
+        # create
+        if params.parent_missing and not self._always_create_workflow_spans:
+            return None
```

Comment:
This line looks like a breaking change, am I getting that wrong?

Reply:
This logic was inside the workflow's form of `_completed_span` before as:

```python
        # If there is no span on the context, we do not create a span
        if opentelemetry.trace.get_current_span() is opentelemetry.trace.INVALID_SPAN:
            return None
```

but now that I have to check a parameter from outside the sandbox, I moved the logic to the outside